src/main/java/org/isf/medicalinventory/service/MedicalInventoryRowIoOperation.java

```diff
@@ -93,4 +93,16 @@ public List<MedicalInventoryRow> getMedicalInventoryRowByInventoryId(int invento
 	public Optional<MedicalInventoryRow> getMedicalInventoryRowById(Integer id) throws OHServiceException {
 		return repository.findById(id);
 	}
+
+	/**
+	 * Return {@link MedicalInventoryRow} for passed param.
+	 *
+	 * @param medicalCode - the medical code.
+	 * @param lotCode - the lot code.
+	 * @return the {@link MedicalInventoryRow} object.
+	 * @throws OHServiceException
+	 */
+	public MedicalInventoryRow getMedicalInventoryRowByMedicalCodeAndLotCode(Integer medicalCode, String lotCode) throws OHServiceException {
```

Comment:
In the method definition (see line 39 below in the next file); the field is defined as a `int` yet the method definition here is an `Integer`.  This causes boxing/unboxing and is a performance hit. My preference is to use primitives instead of objects unless required.

Reply:
Hi @ArnaudFonzam why you marked as resolved this conversation without changes to the code or reason for keep the code uchanged?